tensorboard/plugins/histogram/summary_v2.py

```diff
@@ -21,8 +21,8 @@
 In general, the value of `k` (the number of buckets) will be a constant,
 like 30. There are two edge cases: if there is no data, then there are
 no buckets (the shape is `[0, 3]`); and if there is data but all points
-have the same value, then there is one bucket whose left and right
-endpoints are the same (the shape is `[1, 3]`).
+have the same value, then then all buckets' left and right endpoints are
```

Comment:
While the migration is still in progress, I'd recommend changing this overall file-level docstring to distinguish between the two different formats, since the part here is still relevant to the current histogram_v2 logic.

Basically
- First paragraph w/ just a single line can remain as-is
- Second paragraph is common to both v2 and v3 formats
- Third paragraph (this one) we can split into 2 different paragraphs.  One addressing v2 format (unchanged from existing), one addressing v3 format.

The one about the v3 format we'll want to clarify that the value of `k` is indeed always a constant, so the output is always `[k, 3]`, even in the empty data case (as discussed below).

Reply:
Done.